components/script/dom/servoparser/xml.rs

```diff
@@ -129,7 +129,7 @@ impl<'a> TreeSink for Sink {
             local: name.local,
         };
         let elem = Element::create(name, prefix, &*self.document,
-                                   ElementCreator::ParserCreated);
+                                   ElementCreator::ParserCreated, 1);
```

Comment:
We should add a TODO comment here to add similar APIs to https://github.com/servo/html5ever/pull/240 to the xml5ever parser (which is separate from but similar to html5ever).